docs/platforms/unity/native-support/index.mdx

```diff
@@ -8,27 +8,41 @@ Currently, the Unity SDK offers native support for Android, iOS, Windows, macOS
 
 <Alert>
 
-The setup for the iOS and Android native SDKs happens during build time, with the options defined in the Sentry editor configuration window. Changes to the options object done through [ScriptableOptionsConfiguration](/platforms/unity/configuration/options/) will not affect events from the native layer.
+The Unity SDK captures C# exceptions and does not break any builds targeting Nintendo Switch.
```

Comment:
Do we need this alert?
It's odd to say "And it doesn't break builds"

How about we say:
The Sentry SDK supports C# and Native crashes on Android, iOS, Windows, macOS and Linux.
For Game Consoles, the SDK supports C# errors. While Sentry supports native crashes on consoles, support for that through the Unity SDK is a work in progress. Reach out via GitHub for the platform you're interested in: [PlayStation](https://github.com/getsentry/sentry-unity/issues/2050), [Switch](https://github.com/getsentry/sentry-unity/issues/2052) and [Xbox](https://github.com/getsentry/sentry-unity/issues/2051)